external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -447,7 +440,10 @@ private void shutdown() {
 
     @Override
     public void declareOutputFields(OutputFieldsDeclarer declarer) {
-        kafkaSpoutStreams.declareOutputFields(declarer);
+        RecordTranslator<K, V> translator = kafkaSpoutConfig.getTranslator();
```

Comment:
I agree I think we need a fairly standard way to Tuple->X and X->Tuple.  But really it would just end up looking like

```java
public interface <X> TupleToX {
    public X fromTuple(Tuple t);
}

public interface X XToTuple {
    public Tuple toTuple(X record);
}
```

But I don't know how much code reuse there could be, because X is so different for each library we are going to interface with.